bin/test_client.rb

```diff
@@ -205,6 +206,28 @@ def test_get_notification_by_id_endpoint(client, id, message_type)
   end
 end
 
+def test_get_pdf_for_letter(client, id)
+  response = nil
+
+  # try 15 times with 3 secs sleep between each attempt, to get the PDF
+  for a in 1..15 do
```

Comment:
Using the block syntax is preferred over `for` loops in Ruby, so this line could become `15.times do`